sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/FileSourceStrategySuite.scala

```diff
@@ -529,37 +529,42 @@ class FileSourceStrategySuite extends QueryTest with SharedSparkSession with Pre
   }
 
   test("SPARK-32019: Add spark.sql.files.minPartitionNum config") {
-    withSQLConf(SQLConf.FILES_MIN_PARTITION_NUM.key -> "1") {
-      val table =
-        createTable(files = Seq(
-          "file1" -> 1,
-          "file2" -> 1,
-          "file3" -> 1
-        ))
-      assert(table.rdd.partitions.length == 1)
-    }
+    withSQLConf(
+      SQLConf.FILES_MAX_PARTITION_BYTES.key -> "2MB",
+      SQLConf.FILES_OPEN_COST_IN_BYTES.key -> String.valueOf(4 * 1024 * 1024)) {
+
+      withSQLConf(SQLConf.FILES_MIN_PARTITION_NUM.key -> "1") {
```

Comment:
I think it is okay just to update the slow two tests only in this PR.

Reply:
Done